test_settings.py

```diff
@@ -22,12 +23,13 @@ def root(*args):
 COMPLETION_AGGREGATOR_BLOCK_TYPES = {'course', 'chapter'}
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'default.db',
-        'USER': '',
-        'PASSWORD': '',
-        'HOST': '',
-        'PORT': '',
+        'ATOMIC_REQUESTS': True,
```

Comment:
Is this compatible with how edx-platform works?

Reply:
@jcdyer I can remove it but this is going to be True on all edX Django apps (and is in the devstack). It enables the "wrap all requests in a transaction by default" feature. https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-DATABASE-ATOMIC_REQUESTS